.github/workflows/build_matrix.json

```diff
@@ -5,13 +5,15 @@
         { "sycl": "dpcpp", "sycl-version": "latest", "ubuntu-version": "22.04", "platform": "intel", "build-type": "Debug" },
         { "sycl": "dpcpp", "sycl-version": "latest", "ubuntu-version": "22.04", "platform": "intel", "build-type": "Release" },
         { "sycl": "hipsycl", "sycl-version": "24980221", "ubuntu-version": "20.04", "platform": "nvidia", "build-type": "Debug" },
-        { "sycl": "hipsycl", "sycl-version": "latest", "ubuntu-version": "22.04", "platform": "nvidia", "build-type": "Debug" },
-        { "sycl": "hipsycl", "sycl-version": "latest", "ubuntu-version": "22.04", "platform": "nvidia", "build-type": "Release" }
+        { "sycl": "hipsycl", "sycl-version": "373d3119", "ubuntu-version": "22.04", "platform": "nvidia", "build-type": "Debug" },
```

Comment:
Consider using the same revision for 20.04 and 22.04 as there is no reason (that we know of) to distinguish the two, otherwise LGTM!

Reply:
Both are now using the older of the two revisions.